fab/fabfile.py

```diff
@@ -611,7 +650,7 @@ def preindex_complete():
                 _execute_with_timing(stop_celery_tasks)
             _execute_with_timing(_migrate)
         else:
-            print(blue("No migration required, skipping."))
+            print blue("No migration required, skipping.")
```

Comment:
Why was this changed? (looks like reverting from Python 3 compatible syntax to Python 2 only.